lib/OverlayView/OverlayView.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+export const OverlayView = ({ children }) => {
+  useEffect(() => {
```

Comment:
This effect is going to run on every render of the component. Can we achieve what we need by running the function only once on the component mount?
```
useEffect(() => { ...code }, [])
```

Reply:
Good point, thanks.